include/zephyr/bluetooth/conn.h

```diff
@@ -262,6 +262,139 @@ enum __packed bt_conn_type {
 			   BT_CONN_TYPE_SCO | BT_CONN_TYPE_ISO,
 };
 
+/** Remote channel sounding capabilities for LE connections supporting CS */
+struct bt_conn_le_cs_capabilities {
+	/** Number of CS configurations */
+	uint8_t num_config_supported;
+	/** Maximum number of consecutive CS procedures.
+	 *
+	 * When set to zero, indicates support for both fixed and indefinite
+	 * numbers of CS procedures before termination.
+	 */
+	uint16_t max_consecutive_procedures_supported;
+	/** Number of antennas. */
+	uint8_t num_antennas_supported;
+	/** Maximum number of antenna paths. */
+	uint8_t max_antenna_paths_supported;
+	/** Initiator role. */
+	bool initiator_supported;
+	/** Reflector role. */
+	bool reflector_supported;
+	/** Mode-3 */
+	bool mode_3_supported;
+	/** RTT AA-Only */
+	bool rtt_aa_only_supported;
+	/** RTT Sounding */
+	bool rtt_sounding_supported;
+	/** RTT Random Payload */
+	bool rtt_random_payload_supported;
+	/** RTT AA Only accuracy.
+	 *
+	 * - True: 10ns time-of-flight precision
+	 * - False: 150ns time-of-flight precision (or not supported)
+	 */
+	bool rtt_aa_only_10ns_accurate;
```

Comment:
Maybe we could use an enum (Not supported, 10ns, 150ns) for this and the following member and rename them to `rtt_aa_only_accuracy` and `rtt_sounding_accuracy`?